# Question 1: Area of a Circle
Formula:
area=π*r^2

(Use π = 3.14159)

Input:
Radius ( r = 7 )

In [12]:
radius=int(input("Please input the radius of the circle:"))
pie=3.14
area=pie*(radius ** 2)
print("The area of the circle would be:",area)

Please input the radius of the circle: 7


The area of the circle would be: 153.86


# Question 2: Area of a Rectangle
Formula:
area=length*breadth

Input:
Length = 10
Breadth = 5

In [15]:
length=int(input("Please enter the length of the rectangle:"))
breadth=int(input("Please enter the breadth of the rectangle:"))
area=length*breadth
print("The area of the rectangle is:",area)       

Please enter the length of the rectangle: 10
Please enter the breadth of the rectangle: 5


The area of the rectangle is: 50


# Question 3: Simple Interest
Formula:
SI= (PxRxT)/100 

Input:
Principal (P) = 2000
Rate (R) = 5
Time (T) = 2 years

In [20]:
principal=float(input("Please enter the principal amount:"))
rate=float(input("Please enter the rate of interest:"))
time=float(input("Please enter the duration in years:"))
si=(principal*rate*time)/100
print("The simple interest for your inputs will be:",si)

Please enter the principal amount: 2000
Please enter the rate of interest: 5.5
Please enter the duration in years: 2.5


The simple interest for your inputs will be: 275.0


# Question 6: Find Average of 3 Numbers
Formula:
 

Input:
num1 = 45
num2 = 60
num3 = 75

In [25]:
num1=int(input("Please enter the 1st number:"))
num2=int(input("Please enter the 2nd number:"))
num3=int(input("Please enter the 3rd number:"))
avg=(num1+num2+num3)/3
avg=round(avg)
print("The average of 3 numbers is:",avg)

Please enter the 1st number: 30
Please enter the 2nd number: 20
Please enter the 3rd number: 50


The average of 3 numbers is: 33


# Question 7: Convert Minutes to Seconds
Formula:
Seconds=Minutes*60

Input:
Minutes = 15

In [38]:
minutes=float(input("Please enter minutes to convert:"))
seconds=minutes*60
seconds=round(seconds)
print("The value of",minutes,"minutes in second is:",seconds,"seconds.")

Please enter minutes to convert: 2.5


The value of 2.5 minutes in second is: 150 seconds.


# Question 8: Calculate Total Marks and Percentage
Formula:
Total=m1+m2+m3+m4+m5
Percentage=(Total/500)*100
 

Input:
m1 = 78
m2 = 85
m3 = 69
m4 = 90
m5 = 74

In [41]:
m1=int(input("Please enter the value of m1:"))
m2=int(input("Please enter the value of m2:"))
m3=int(input("Please enter the value of m3:"))
m4=int(input("Please enter the value of m4:"))
m5=int(input("Please enter the value of m5:"))
total=m1+m2+m3+m4+m5
percentage=(total/500)*100
print("Total value is:",total)
print("percentage of total value is:",percentage,"%")

Please enter the value of m1: 78
Please enter the value of m2: 85
Please enter the value of m3: 69
Please enter the value of m4: 90
Please enter the value of m5: 74


Total value is: 396
percentage of total value is: 79.2 %


# Question 9: Convert Kilometers to Meters
Formula:
Meters=Kilometers*1000

Input:
Kilometers = 6.5

In [46]:
kilometers=float(input("Please enter the value in kilometer:"))
meters=kilometers*1000
print("The value of",kilometers,"kilometers in meter is:",meters)

Please enter the value in kilometer: 6.5


The value of 6.5 kilometers in meter is: 6500.0


# Question 10: Volume of a Cube
Formula:
Volume=Side**3

Input:
Side = 4

In [51]:
side=int(input("Please enter the value of the side of a cube:"))
volume=side**3
print("The volume of a cube is:",volume)

Please enter the value of the side of a cube: 4


The volume of a cube is: 64


# Question 11: Calculate Speed
Formula:
Speed=Distance/Time

Input:
Distance = 150 km
Time = 3 hours

In [60]:
dist=float(input("Please enter the kilometer you have travelled:"))
time=float(input("Please enter the time you have taken to travel in hours:"))
speed=dist/time
speed=round(speed)
print("Your speed was:",speed,"kilometers/hour.")

Please enter the kilometer you have travelled: 150
Please enter the time you have taken to travel in hours: 3


Your speed was: 50 kilometers/hour.
